### DISCUSSION TITLE
How to pull dashboards in a SPECIFIED folder...?

### DISCUSSION THREAD

**Discussion by kingDaddyFreshness** [grafana/grizzly#229] [Q&A]
Hello all,

in the documentation I see that you can pull all dashboards or a specific dashboard - and that you can pull all folders or a specific folder....  Was wondering if there was anyway to pull all dashboards that exist within a PARTICULAR/SPECIFIED folder?

I'm sure I could use pull each specific dashboard that was in a folder if I new all the uids of the dashboards within a folder...  but was hoping there was some nice command line trick?

Example:  grr -t "DashboardFolder/SpecificFolder" -t "Dashboard/*"

Does such a thing already exist?

Thanks!

-john

**Comment by malcolmholmes** [comment]
This does not exist. It does make a lot of sense as a feature, but I haven't worked out how I would implement it - i.e. what syntax would make sense - it is an odd scenario - where relationships between folders and dashboards matter, and these are separate resources in Grizzly.

**Comment by amisidutravail** [comment]
Hello @malcolmholmes , has this situation changed ? Can you select or pull only the dashboard only for 1 folder ?

I want to download a folder full of dashboard, to be able to apply something only on thoses ones.
Is there anyway to do it with the client or do I have to  use a script to check for parent in the JSON for instance ?


Regards

amisidutravail

**Comment by gespi1** [comment]
pulling by tags would be great as well

**Comment by DerkieH** [comment]
This would be a nice feature, +1
